docs/source/en/model_doc/audioflamingo3.md

```diff
@@ -0,0 +1,159 @@
+# AudioFlamingo3
+
+## Overview
+
+AudioFlamingo3 (AF3) is a fully open large audio–language model designed for robust understanding and reasoning over speech, environmental sounds, and music. AF3 pairs a Whisper-style audio encoder with a causal language model and performs replace-in-place audio–text fusion: the processor aligns post-pool audio frames to a dedicated placeholder token (written as `<sound>` in code) and the model replaces those token slots with projected audio embeddings during the forward pass.
+
+Highlights:
+
+- Unified audio encoder across speech, sound, and music.
+- Long-audio support via windowing and post-pool alignment (up to about 10 minutes by default).
+- Deterministic fusion that preserves sequence length by replacing `<sound>` tokens with audio embeddings.
+- Production-oriented processing flow with batch safety and strict shape checking.
+
+### Paper
+
+[Audio Flamingo 3](https://arxiv.org/abs/2507.08128): Advancing Audio Intelligence with Fully Open Large Audio Language Models  
+A. Goel, S. Ghosh, J. Kim, S. Kumar, Z. Kong, S. Lee, C.-H. H. Yang, R. Duraiswami, D. Manocha, R. Valle, B. Catanzaro  
+NVIDIA and University of Maryland  
+Project: https://research.nvidia.com/labs/adlr/AF3/
+
+## Quickstart
+
+```python
+from transformers import AudioFlamingo3Processor, AudioFlamingo3ForConditionalGeneration
+import librosa, torch
+
+
+def load_audio(p, sr=16000):
+    y, _ = librosa.load(p, sr=sr, mono=True)
+    if y.size:
+        dmin, dmax = y.min(), y.max()
+        y = (2 * y / (abs(dmax) or 1.0) - 1.0) if dmin >= 0 else y / (max(abs(dmax), abs(dmin)) or 1.0)
+    return y.astype("float64")
+
+
+device = "cuda" if torch.cuda.is_available() else "cpu"
+processor = AudioFlamingo3Processor.from_pretrained("nvidia/audio-flamingo-3")
+model = AudioFlamingo3ForConditionalGeneration.from_pretrained("nvidia/audio-flamingo-3").to(device).eval()
+
+texts = ["Transcribe the input speech.", "Describe the music in details."]
+audios = [load_audio("audio_1.wav"), load_audio("audio_2.wav")]
+inputs = processor(texts, audios, padding_side="left", tensor_type="pt").to(device)
+
+output_ids = model.generate(**inputs, max_new_tokens=2048, do_sample=False)
+texts = processor.batch_decode(output_ids, skip_special_tokens=True)
+texts = [text.split("\nassistant\n")[-1] for text in texts]
+print(texts)
+```
+
+Tips:
+
+* Left padding is recommended for batched generation with mixed prompt lengths.
+* The processor returns conversation-formatted text; the snippet above keeps only the assistant’s reply.
```

Comment:
Remove? With left padding as default + chat_template removed from within the processor